assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_attribution_drift.py

```diff
@@ -124,22 +124,49 @@ def compute_ndcg_and_write_to_mltable(baseline_explanations, production_explanat
     save_spark_df_as_mltable(spark_data, feature_attribution_data)
 
 
-def configure_data(data):
-    """Convert pySpark.Dataframe to pandas.Dataframe and sort the data.
+def configure_signal_metrics_data(data):
+    """Convert pySpark.Dataframe to pandas.Dataframe and sort the data for signal metrics table.
 
     :param data: feature importances with their corresponding feature names
     :type data: pySpark.Dataframe
-    :return: the sorted pandas feature importances data with the row count dropped and the number of rows
+    :return: the sorted pandas global feature importances data with the row count dropped and the number of rows
     :rtype: tuple of pandas dataframe and number
     """
     df = read_mltable_in_spark(data).toPandas()
+    # Remove local feature importance sample
+    df.dropna(subset=[constants.METRIC_NAME_COLUMN], inplace=True)
```

Comment:
so this is temporary right ? while samples is not supported yet ?

Reply:
Since the local feature importance sample table and signal metrics (global feature importance) tabls are sent to this component as a franken-table this method is extracting the signal metrics table so it removes the local importance sample. configure_local_importance_sample_data() does the opposite and extracts the local importance sample table.